components/sup/src/command/start.rs

```diff
@@ -84,7 +84,7 @@ pub fn package() -> Result<()> {
         return Err(sup_error!(Error::RootRequired));
     }
 
-    match Package::load(gconfig().package(), None) {
+    match Package::load(gconfig().package(), Some(&*FS_ROOT_PATH)) {
```

Comment:
I traced this through down to `core::package::PackageInstall::resolve_package_install()` and we look okay here. Long term we might want to consider changing the `Option<&Path>` to a `&Path` but this is more than fine right now.